__init__.py

```diff
@@ -24,16 +24,16 @@ def name():
 def description():
     return "Edit toolbar for PostGIS topology primitives (ISO SQL/MM based)"
 def version():
-    return "0.2.2"
+    return "0.3.1"
 def icon():
     return "icons/topoedit.png"
 def qgisMinimumVersion():
-    return "1.8"
+    return "3.0"
```

Comment:
Does this mean we're dropping support for 1.8 with this change ?

Reply:
The porting guidelines suggested keeping plugins for 2 and 3 seperated in different branches. Since it's both different Qt-versions, different API-versions and different python versions I figured it's a good idea. Only one of them will show in the plugins listing in the client depending on version.